README.md

```diff
@@ -1,24 +1,27 @@
 # Homebrew
-Features, usage and installation instructions are [summarised on the homepage](http://brew.sh).
+**Features, usage and installation instructions are [summarised on the homepage](http://brew.sh).**
 
 ## What Packages Are Available?
-1. Type `brew search` for a list.
-2. Or visit [braumeister.org](http://braumeister.org) to browse packages online.
-3. Or use `brew search --desc <keyword>` to browse packages from the command line.
+1. Use `brew search` on the command line for a list of packages.
+2. Use `brew search --desc <keyword>` to browse packages from the command line.
+3. Alternatively, you can visit [braumeister.org](http://braumeister.org) to browse packages online.
 
 ## More Documentation
-`brew help`, `man brew` or check [our documentation](https://github.com/Homebrew/brew/tree/master/share/doc/homebrew#readme).
+Use `brew help` or `man brew` on the command line. Additionally, check [our documentation](https://github.com/Homebrew/brew/tree/master/share/doc/homebrew#readme).
 
 ## Troubleshooting
-First, please run `brew update` and `brew doctor`.
 
-Second, read the [Troubleshooting Checklist](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/Troubleshooting.md#troubleshooting).
+Before creating an [issue](https://github.com/Homebrew/brew/issues/), please run `brew update` twice and `brew doctor` once from the command line. This should resolve most issues. 
 
-**If you don't read these it will take us far longer to help you with your problem.**
+If the issue persists, read the [Troubleshooting Checklist](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/Troubleshooting.md#troubleshooting).
+
+**If you don't read over these, it will take us far longer to help you with your problem.**
```

Comment:
I'm not sure we want to expand on this section. The longer we make it, the fewer people will actually read it. I think it's fine adding a “(twice)” to “please run `brew update`”, but otherwise I'd prefer to keep this section as-is. Everything else is already covered in the linked Troubleshooting Checklist.

Also, if a link to creating a new issue is the first thing people see in this section, they will most likely stop reading the additional instructions and just go ahead creating an issue. Sadly, people are often enough like that and the link just encourages this behavior.